api/app/Providers/AppServiceProvider.php

```diff
@@ -32,6 +34,27 @@ public function boot()
             'personalExperience' => \App\Models\PersonalExperience::class,
             'workExperience' => \App\Models\WorkExperience::class,
         ]);
+
+        // DB::whenQueryingForLongerThan(50, function (Connection $connection, QueryExecuted $event) {
+        //     // Notify development team...
+        //     Log::debug(
+        //         [
+        //             'sql' => $event->sql,
+        //             'bindings' => $event->bindings,
+        //             'milliseconds' => $event->time,
+        //         ]
+        //     );
+        // });
```

Comment:
As seen in Laravel docs, did not work well as it doesn't output much info about the entire event, rather, after it hits some time cutoff it outputs the last statement to push it over the time. Even if that statement didn't take that long to execute

![Screenshot 2024-01-17 170312](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/d48fa108-d433-4cd2-997e-fcaeb4815ed5)

Reply:
Too bad.  It's a nice looking facade.